pages/view/_iscnId.vue

```diff
@@ -178,7 +178,14 @@
             :label="$t('iscn.meta.owner')"
             class="mb-[12px]"
           >
-            <Label :text="owner" tag="div" preset="p6" />
+            <Link
+              :class="[
+                'text-[14px]',
+                'break-all',
+              ]"
+              :href="`/search?q=${owner}`">
```

Comment:
can we use name route instead of URL, and use explicit query for each field instead of query All (q)?

Reply:
ie https://github.com/likecoin/app-like-co/blob/main/pages/search/_keyword.vue#L164 this function `queryISCNByField` only support `keyword` now but it support owner/stakholder query on API level, please help implement the support of these field and use this function instead of `q`